src/w3c/templates/cgbg-sotd.js

```diff
@@ -1,6 +1,6 @@
 import hyperHTML from "hyperhtml";
 
-export default conf => {
+export default (conf, opts) => {
```

Comment:
Intentionally didn't use object spread because I think we'll migrate more and more options here that will generate too much lines.

Reply:
makes sense.